src/Support/SelectFields.php

```diff
@@ -357,4 +417,9 @@ public function getRelations(): array
     {
         return $this->relations;
     }
+
+    public function getResolveInfo(): ResolveInfo
```

Comment:
This isn't necessary: ResolveInfo is solely here consumed by the class; any query/mutation resolver it as a separate argument anyway